docs/sql-ref-syntax-ddl-create-table-datasource.md

```diff
@@ -29,14 +29,14 @@ The `CREATE TABLE` statement defines a new table using a Data Source.
 CREATE TABLE [ IF NOT EXISTS ] table_identifier
     [ ( col_name1 col_type1 [ COMMENT col_comment1 ], ... ) ]
     USING data_source
-    [ OPTIONS ( key1=val1, key2=val2, ... ) ]
+    [ OPTIONS ( ( key1=val1, key2=val2, ... ) | ( key1 val1, key2 val2, ... ) ) ]
```

Comment:
I guess we can use `[ ]`to make the `=` optional?
```
[ OPTIONS ( key1 [ = ] val1 [  , ...  ]  )  ]
```

Reply:
Agree, I don't have strong arguments against your version. I would argue that the my version is more readable for a documentation.

Reply:
I am OK either way. As long as we keep these consistent for all the ddl statements, I am fine.

Reply:
I ultimately went with your proposal 👍  My version didn't feel that readable and concise anymore when using it in the various places